### PR TITLE
Get interfaces

### PR DESCRIPTION
**Pull request by andreamlin**
When service config is not given, just look at `model.getInterfaces()`. When service config is given, look at `model.getApis()`.

This blocks https://github.com/googleapis/gapic-generator/pull/2532

### CODE REVIEW

**Comment by andreamlin** [comment]
PTAL

File: src/main/java/com/google/api/codegen/gapic/ProtoModels.java [hunk 1]
```diff
@@ -27,8 +29,19 @@
...
   private ProtoModels() {}
 
+  /** Gets the interfaces for the apis in the model. */
+  public static List<Interface> getInterfaces(
+      Model model, @Nullable GapicProductConfig productConfig) {
+    if (productConfig == null || !productConfig.getProtoParser().isProtoAnnotationsEnabled()) {
+      return getInterfacesFromServiceConfig(model);
+    }
+
+    // Assume valid service config if proto annotations parsing is not enabled.
```
Comment (michaelbausor) [inline src/main/java/com/google/api/codegen/gapic/ProtoModels.java]: Won't we end up here if proto annotation parsing *is* enabled?

Comment (andreamlin) [reply src/main/java/com/google/api/codegen/gapic/ProtoModels.java]: Oh yes, fixed comment.

**Comment by andreamlin** [comment]
Fixed wrong comment; PTAL

**Pull request merged**